set.mm

```diff
@@ -467676,6 +468012,32 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
       YKZVWPUYHVVJXUSUYDUYIUYMVUGEUYKUBVWTUYERYTYLYMUYFUYG $.
   $}
 
+  ${
+    $d U x $.  $d W x $.  $d Y x $.
+    qusdimsum.x $e |- X = ( W |`s U ) $.
+    qusdimsum.y $e |- Y = ( W /s ( W ~QG U ) ) $.
+    $( Let ` W ` be a vector space, and let ` X ` be a subspace.  Then the
+       dimension of ` W ` is the sum of the dimension of ` X ` and the
+       dimension of the coset of ` X ` .  First part of theorem 5.3 in [Lang]
```

Comment:
Not a "coset", but a "factor space" (or a "quotient space" if you want to be consistent with your earlier comment, but Lang uses "factor space").